_posts/2020-05-29-homebrew-2.3.0.md

```diff
@@ -0,0 +1,37 @@
+---
+title: 2.3.0
+author: MikeMcQuaid
+---
+Today I'd like to announce Homebrew 2.3.0. The most significant changes since 2.2.0 are GitHub Actions CI usage, fetch before installation, Docker image improvements and the deprecation of `brew install` from URLs.
```

Comment:
```suggestion
Today I'd like to announce Homebrew 2.3.0. The most significant changes since 2.2.0 are GitHub Actions CI usage, fetching resources before installation, Docker image improvements and the deprecation of `brew install` from URLs.
```

"fetch before installation" can be made less ambiguous by being just a little more explicit. I understood what you meant here because I'm familiar with the change but anyone else would have to read further to understand.

I resisted sneaking an Oxford comma into this suggestion 😇